src/main/kotlin/com/devfloor/untitled/article/application/ArticleFacade.kt

```diff
@@ -12,8 +16,53 @@ class ArticleFacade(
     private val articleService: ArticleService,
     private val articleHashtagService: ArticleHashtagService,
     private val articleFavoriteService: ArticleFavoriteService,
-    private val articleOptionService: ArticleOptionService
+    private val articleOptionService: ArticleOptionService,
+    private val optionService: OptionService,
+    private val hashtagService: HashtagService
 ) {
+    @Transactional
+    fun create(articleRequest: ArticleRequest, user: User): ArticleResponse {
+
+        val article = articleService.create(articleRequest.toArticleWithUser(articleRequest, user))
```

Comment:
밑의작성한 리뷰를 적용하게 된다면 이 코드가 아래와같이 변경될 것 같아요 !
```suggestion
        val article = articleService.create(Article.of(articleRequest, user))
```